### PR TITLE
fix lora modules_to_save issue

### PR DESCRIPTION
**Pull request by pacman100**
### What does this PR do?
1. Currently, `modules_to_save` aren't working when using LoRA. This PR fixes it. https://github.com/huggingface/peft/issues/341#issuecomment-1516013491

### CODE REVIEW

File: tests/testing_common.py [hunk 1]
```diff
@@ -68,7 +68,7 @@
 class ClassInstantier(OrderedDict):
     def __getitem__(self, key, *args, **kwargs):
         # check if any of the kwargs is inside the config class kwargs
-        if any([kwarg in self[key][1] for kwarg in kwargs]):
+        if any(kwarg in self[key][1] for kwarg in kwargs):
```
Comment (younesbelkada) [inline tests/testing_common.py:71]: Curious why this is needed?

**Review by younesbelkada — APPROVED** [review]
Thanks a lot for fixing! 🔥 
Left one question, otherwise LGTM

File: tests/testing_common.py (hunk 1 above)
Comment (pacman100) [reply tests/testing_common.py:71]: ruff raises quality issue.

Comment (younesbelkada) [reply tests/testing_common.py:71]: Ok I see, I think it's ok to merge it as it is,

**Pull request merged**